snap/config/security-secret-store/vault-config.json.in

```diff
@@ -0,0 +1,21 @@
+listener "tcp" { 
```

Comment:
Where did this file originate? Is it based on logic within the vault Dockerfile?

Reply:
It's from here : https://github.com/edgexfoundry/security-secret-store/blob/master/local-tls.json
Yes the host names and paths such have been modified from what's there that's why I copied the file.